File: python/input_output.py
import matplotlib.pyplot as plt
import numpy as np

def lectureImage(chemin):
	return plt.imread(chemin)

def AfficherImg(img):
	plt.axis("off")
	plt.imshow(img, interpolation="nearest")
	plt.show()

def saveImage(img, nom="image1.png"):
	plt.imsave(nom, img)

File: matlab/input_output.m
function Img = lectureImage(chemin)
    Img = imread(chemin);
end

function AfficherImg(img)
    imshow(img, []);
end

function saveImage(img, nom)
    if nargin < 2
        nom = 'image1.png';
    end
    imwrite(img, nom);
end
